.github/workflows/crawler.yml

```diff
@@ -3,10 +3,21 @@
 name: Crawler
 
 on:
+  push:
+    branches: [ master, main ]
+    tags: [ v* ]
+  pull_request:
   schedule:
   - cron: '0 14 * * *'
   workflow_dispatch:
+    inputs:
+      args:
+        description: Extra args to pass to crawler.rb script
+        required: false
+        default: 'latest'
```

Comment:
Is this a default value for `mode`? We don't have `status-smg-3GPP-latest` data source. For the latest documents we should use `status-smg-3GPP`. So default value should be empty.

Reply:
Updated to 'checkbox' to validate input (checked - force, unchecked - regular)